library/src/main/java/com/novoda/downloadmanager/DownloadManagerBuilder.java

```diff
@@ -66,9 +68,35 @@ public static DownloadManagerBuilder newInstance(Context context, Handler callba
         FileSizeRequester fileSizeRequester = new NetworkFileSizeRequester(httpClient, requestCreator);
         FileDownloader fileDownloader = new NetworkFileDownloader(httpClient, requestCreator);
 
-        NotificationChannelCreator notificationChannelCreator = new DownloadNotificationChannelCreator(context.getResources());
+        NotificationCustomizer<DownloadBatchStatus> notificationCustomizer = new NotificationCustomizer<DownloadBatchStatus>() {
 
-        NotificationCreator<DownloadBatchStatus> downloadBatchNotification = new DownloadBatchNotification(context, notificationIcon);
+            private static final boolean NOT_INDETERMINATE = false;
+
+            @Override
+            public Notification customNotificationFrom(NotificationCompat.Builder builder, DownloadBatchStatus payload) {
+                DownloadBatchTitle downloadBatchTitle = payload.getDownloadBatchTitle();
+                int percentageDownloaded = payload.percentageDownloaded();
+                int bytesFileSize = (int) payload.bytesTotalSize();
+                int bytesDownloaded = (int) payload.bytesDownloaded();
+                String title = downloadBatchTitle.asString();
+                String content = percentageDownloaded + "% downloaded";
+
+                return builder
+                        .setProgress(bytesFileSize, bytesDownloaded, NOT_INDETERMINATE)
+                        .setSmallIcon(notificationIcon)
+                        .setContentTitle(title)
+                        .setContentText(content)
+                        .build();
+
+            }
+        };
+        NotificationConfig<DownloadBatchStatus> notificationConfig = new NotificationConfig<>(
+                context,
```

Comment:
I'm assuming that this should match other areas in here? If so it should use the `applicationContext` defined above 😸

Reply:
`NotificationConfig` calls `getApplicationContext` on whichever context gets passed in so it doesn't matter